src/pages/welcome.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled, { ThemeProvider } from 'styled-components/macro'
+import { theme } from 'theme'
+import logo from 'images/logo.png'
+
+const localTheme = ({ colors, sizes, fontFamily }: typeof theme) => ({
+  background: colors.background,
+  headerBgColor: colors.primary,
+  headerFontColor: colors.backgroundLight,
+  headerFontSize: sizes.question,
+  botBubbleColor: colors.backgroundLight,
+  botFontColor: colors.text,
+  userBubbleColor: colors.secondaryLight,
+  userFontColor: colors.backgroundLight,
+  fontFamily,
+  colors,
+  sizes
+})
+
+const Wrapper = styled('div')`
+  h1 {
+    color: ${props => props.theme.colors.primary};
+    font-size: calc(${props => props.theme.sizes.buttonText} * 1.5);
+    font-family: ${props => props.theme.fontFamily};
+    padding: calc(${props => props.theme.sizes.buttonText} * 0.75);
+    font-weight: 500;
+    display: flex;
```

Comment:
H1s (and H2s, P, etc) should normally not be flex components, as they're not meant to have any non-inline children

Reply:
looking at this, you can probably be fine with removing all the display: flex inside text elements

Reply:
if you want centering, then use `text-align: center`